webapp/src/DepCard.js

```diff
@@ -16,8 +16,9 @@ class DepCard extends PureComponent {
 
   blockerCount(nodes) {
     var count = 0;
-    for (var index in (this.props.dependencies || [])) {
-      if (true) {
+    var dependencies = this.props.dependencies || [];
+    for (var index in dependencies) {
+      if (Object.prototype.hasOwnProperty.call(dependencies, index)) {
```

Comment:
I looked at the link you posted now, missed that first... However, I don't think we have to be worried about Array.prototype being override, in that case, we should stop using whatever lib is doing that.